packages/vite/src/node/server/middlewares/decodeURI.ts

```diff
@@ -0,0 +1,10 @@
+import { Connect } from 'types/connect'
```

Comment:
I think we can use a type import here

```suggestion
import type { Connect } from 'types/connect'
```

Please test if the Vite project support that (run `yarn build` locally)